beetmoverscript/script.py

```diff
@@ -82,6 +89,27 @@
     await retry_download(context=context, url=source, path=abs_file_path)
     await retry_upload(context=context, destinations=destinations, path=abs_file_path)
 
+    if context.task['payload']['update_manifest']:
+        enrich_balrog_manifest(context, abs_file_path, locale)
```

Comment:
true. I guess my concern is that 'update_manifest' will be True for the whole signing based beetmover task. So each artifact in the task_to_beetmove will be added to the manifest. We will have to be careful if we ever add an artifact in the signing task, e.g. checksums